sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1216,6 +1216,16 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val FILE_COMMIT_STAGING_DIR =
```

Comment:
Since always SQL related, so add this config in SQL part, although it's used in core part